frontend/integration-tests/tests/kubevirt/utils.ts

```diff
@@ -114,3 +114,16 @@ export const waitForCount = (elementArrayFinder, targetCount) => {
   };
 };
 
+export function exposeService(exposeSrv: Set<string>) {
```

Comment:
Just a nitpick, I'd prefer exposeServices as the method takes Set

Reply:
Changed.